src/external/song.ts

```diff
@@ -80,12 +88,13 @@ export const getAnalysesByStudy = async (url: string, studyId: string): Promise<
 };
 
 export const getAnalysesById = async (
-  url: string,
+  dataCenterId: string,
   studyId: string,
   analysisId: string,
 ): Promise<SongAnalysis> => {
+  const dataCenter = await getDataCenter(dataCenterId);
   const analysesUrl = urljoin(
```

Comment:
hm I see now this flat casing of `urlJoin` is in more than just that one file. disregard my previous comment